build.gradle

```diff
@@ -264,6 +264,7 @@ project('spring-integration-core') {
 		compile "org.springframework:spring-tx:$springVersion"
 		compile "org.springframework.retry:spring-retry:$springRetryVersion"
 		compile ("io.projectreactor:reactor-core:$reactorVersion", optional)
+		compile ("io.projectreactor:reactor-stream:$reactorVersion", optional)
```

Comment:
After this, it looks redundant to have `reactor-core` here.
Isn't it?
BTW, having you change in `master` we won't be able to merge your work properly